Configuration/Eras/python/Modifier_run2_ECAL_2017_cff.py

```diff
@@ -0,0 +1,3 @@
+import FWCore.ParameterSet.Config as cms
+
+run2_ECAL_2017 =  cms.Modifier()
```

Comment:
once added to Run2_2017, it will also show up in Run2_2018.
The phase-2 settings should be done separately. Given planned electronics upgrades, it's unclear to me which thresholds are reasonable there.